### ISSUE TITLE
Animations usage tutorial

### ISSUE THREAD

**Issue by arnoldozippiex1** [alexvasilkov/GestureViews#126]
is there any simple tutorial to show how to use animations?
i have used GestureViews to create a comic book and want to zoom to different parts of the page.
any help will be regarded.

**Comment by alexvasilkov** [comment]
There are no tutorial unfortunately, but you can explore sample app's code, I tried to split it to small self-contained examples and provided comments in the code.

I don't know your use case, but you can probably look into `GestureFrameLayout` usage example ([layout viewer](https://github.com/alexvasilkov/GestureViews/tree/master/sample/src/main/java/com/alexvasilkov/gestures/sample/ex/layout/viewer)) or use an images-grid-to-pager animation, similar to [recycler to pager](https://github.com/alexvasilkov/GestureViews/tree/master/sample/src/main/java/com/alexvasilkov/gestures/sample/ex/list/recycler) sample.

**Comment by arnoldozippiex1** [comment]
thanks for your grate library and your answer.
I managed to use basic animation. but I have another question. how can i setInterpolator for animations?
thanks again.

**Comment by alexvasilkov** [comment]
There is no easy way to do so, you can try to change field under `ViewPositionAnimator.positionScroller.interpolator` through reflection, though it is not really recommended.

Why do you need to change the interpolator, doesn't `AccelerateDecelerateInterpolator` work for you?

**Comment by alashow** [comment]
Spring animations would be an awesome addition

**Comment by alexvasilkov** [comment]
@alashow Spring animations would be much harder to integrate into existing code, it is not like setting a new interpolator. How exactly you think it can be used in the library?
I can see one use case for physics based animation for exit gesture, but do you think there are other places where it can be used as well?

**Comment by alashow** [comment]
@alexvasilkov I don't know much animations so I thought it could be used as an interpolator? That bouncy at the end of a motion. Like in [here](http://facebook.github.io/rebound-js/examples/) or photo viewer opening/closing in Twitter Android app.

**Comment by alexvasilkov** [comment]
I tried Twitter's official Android app, can't see they use spring animations.

Android's new Spring animation is a separate library (https://developer.android.com/guide/topics/graphics/spring-animation), it is not a standalone interpolator. `OvershootInterpolator` is close to what you said, but I tried it works quite ugly.

So I'm still unsure how we can use spring animations. I also don't want to depend on the springs library, so it should be optional anyway.
